server/datasources.json

```diff
@@ -0,0 +1,10 @@
+{
+  "memory": {
```

Comment:
`memory` in this case is really intended to be using memory to enable quick lookup of ACLs etc.
The other datasource is called `db` back by memory by default and configurable to use other DBs